vendor/github.com/openshift/library-go/pkg/cloudprovider/external.go

```diff
@@ -23,8 +23,7 @@ func IsCloudProviderExternal(platformStatus *configv1.PlatformStatus, featureGat
 	}
 	switch platformStatus.Type {
 	case configv1.AWSPlatformType,
-		configv1.GCPPlatformType,
-		configv1.VSpherePlatformType:
```

Comment:
it looks like we need to change some tests in https://github.com/openshift/machine-config-operator/blob/26eaa2c19b8d0434de7eca9f2c3806ef7fc9f05a/pkg/controller/template/render_test.go#L26

i.e. NutanixPlatformType "" -> external (2 instances of this)
VSpherePlatformType vsphere -> external

does that seem correct?

Reply:
Yeah that seems correct, the nutanix folks introduced a CCM recently and want to initialize their machines using the new CCM, or at least that's my understanding, I'll double check

Reply:
@JoelSpeed I think Nutanix CCM’s node-controller will prep the node and then remove the taint "node.cloudprovider.kubernetes.io/uninitialized".